lib/repl.js

```diff
@@ -223,6 +229,29 @@ module.paths = CJSModule._nodeModulePaths(module.filename);
 const writer = (obj) => inspect(obj, writer.options);
 writer.options = { ...inspect.defaultOptions, showProxy: true };
 
+// Converts static import statement to dynamic import statement
+const toDynamicImport = (codeLine) => {
+  let dynamicImportStatement = '';
+  let moduleName = '';
+  const toCamelCase = (str) => str.replace(/[-_](\w)/g, (_, c) => c.toUpperCase());
+  const ast = acornParse(codeLine, { sourceType: 'module', ecmaVersion: 'latest' });
+  acornWalk.ancestor(ast, {
+    ImportDeclaration: (node) => {
+      const importedModules = node.source.value;
+      const importedSpecifiers = node.specifiers.map((specifier) => specifier.local.name);
+      if (importedSpecifiers.length > 1) {
+        moduleName = `{${importedSpecifiers.join(',')}}`;
+      } else {
+        const formattedSpecifiers = importedSpecifiers.length ? ArrayPrototypeToString(importedSpecifiers) : '';
+        moduleName = toCamelCase(formattedSpecifiers || importedModules);
+      }
+      dynamicImportStatement += `const ${moduleName} = await import('${importedModules}');`;
```

Comment:
```suggestion
      if (node.specifiers.length === 0) {
        dynamicImportStatement += `await import(${JSONStringify(node.source.value)});`
      } else {
        const toObjectNotation = ({ local, imported }) => local.name === imported.name ? local.name : `${imported.name}: ${local.name}`;
        const importNames = node.specifiers.length === 1 ?
          toObjectNotation(node.specifiers[0]) :
          ArrayPrototypeJoin(ArrayPrototypeJoin(node.specifiers, toObjectNotation), ', ');
        dynamicImportStatement += `const ${importNames} = await import('${importedModules}');`;
      }
```

Reply:
Looks like this bombs for `import * as name from "module-name";`?